Automattic/vip-go-mu-dev
# Add repo and branch switching

## Description

Added a way to swap repos/branches in the lando environment. You can clone and checkout a specific branch or clone and then use regular git commands from wp-content for the repo you pulled.

## Steps to test

1. Apply PR
2. Pick a VIP repo with multiple branches. Make sure to note the SSH path to their repo(e.g.: `git@github.com:org/repo.git`) and branches. 
3. `lando vip-switch <ssh path>`
4. It should apply for the site as it is on `master`
5. `lando vip-switch <ssh path> -b <non-master-branch>`
6. It should apply for the site as it is on the branch you specified.
7. `lando vip-switch <ssh path> --branch <non-existant-branch>`
8. It should error and nothing should change.
